.github/labels.yml

```diff
@@ -55,18 +61,27 @@
 - name: "Type: 🧹 chore"
   description: "Routine tasks, maintenance, or non-user-facing changes."
   color: "FAD8C7"
+- name: "Type: ❓ question"
+  description: "Not an classical issue. Customer's question."
+  color: "C7FAD8"
 
 # Status
+- name: "Status: 🖐 opened"
+  description: "Task currently opened."
+  color: "45DC35"
```

Comment:
Why is it necessary? Because there is an issue status (closed, opened)

Reply:
It important for statistics and some future automatisation. Why not?